cert/metrics.go

```diff
@@ -94,56 +72,28 @@ var (
 			Name:      "spec_next_wake_timestamp",
 			Help:      "The epoch value of when this spec will next awaken to perform checks",
 		},
-		[]string{"spec_path"},
-	)
-
-	// ActionAttemptedCount counts actions taken by a spec
-	ActionAttemptedCount = prometheus.NewCounterVec(
-		prometheus.CounterOpts{
-			Namespace: metricsNamespace,
-			Name:      "action_attempted_count",
-			Help:      "Number of times a spec has taken action",
-		},
-		[]string{"spec_path", "change_type"},
-	)
-
-	// ActionFailedCount counts failed actions taken by a spec
-	ActionFailedCount = prometheus.NewGaugeVec(
-		prometheus.GaugeOpts{
-			Namespace: metricsNamespace,
-			Name:      "action_failed_count",
```

Comment:
This metric seems especially useful. I understand that it doesn't fit the new architecture, but how will we now monitor when certmgr's actions fail?

Reply:
This metric is actually redundant; we had two metrics, 'write to disk' and 'action notification failed'.

With the new architecture, we still have the 'store failed'- 'store' just now includes "write to disk and trigger an action" if that is the backend in use.

Said another way: for alerting, we only have to track one metric now- not two.  We still have visibility of failure.

Reply:
Got it, thanks for the explanation.